uint8 ZONE
#ZONE_OPEATING                 = 0;      // Khu vuc van hanh. Di chuyen theo toc do cai dat
#ZONE_HAZARD_OPERATING         = 1;      // Khu vuc van hanh co rui ro. Toc do toi da (trung binh) cho phep la 0.6 m/s
#ZONE_RESTRICTED               = 2;      // Khu vuc nguy hiem. Toc do toi da cho phep la 0.3 m/s (Su dung tai vị tri vao sạc hoac nhung vi tri co khong gian hep)
#ZONE_LOAD_TRANSFER            = 3;      // Khu vuc chuyen tai. Toc do toi da cho phep la 0.3 m/s (Su dung tai nhung vi tri trao doi tai hoac co tuong tac)
#ZONE_CONFINED                 = 4;      // Khu vuc dac biet, cam nguoi khong co quyen trong khu vuc nay 
#ZONE_FORBIDDEN                = 5;      // Khu vuc cam. Khong cho robot di chuyen vao khu vuc nay